docs/source/getting_started/tutorials.md

```diff
@@ -38,12 +38,12 @@ Learn how to [run runtime-specific pipelines on Kubeflow Pipelines](https://gith
 #### Run generic pipelines on Apache Airflow
 
 Learn how to [run generic pipelines on Apache Airflow](https://github.com/elyra-ai/examples/tree/main/pipelines/run-generic-pipelines-on-apache-airflow). This tutorial requires an Apache Airflow deployment in a local environment or on the cloud.
-If you want to run generic pipelines in Airflow >= 2.7.0, you have to use Elyra 4. In Elyra 4, generic pipeline support for Airflow 1 is removed.
+Elyra 4 supports Apache Airflow 2.7.0+ and 3.x with automatic version detection. Generic pipeline support for Airflow 1.x is removed in Elyra 4.
```

Comment:
Could we leave this similar to the left? It's ok to say something is removed on a specific version, but the overall version independent phrase let it be true independent of the version being released (e.g. when we release version 4.1 or 5)